src/hotspot/share/gc/g1/g1CollectionSetCandidates.cpp

```diff
@@ -134,7 +134,7 @@ int G1CSetCandidateGroup::compare_gc_efficiency(G1CSetCandidateGroup** gr1, G1CS
   }
 }
 
-int G1CSetCandidateGroup::compare_reclaimble_bytes(G1CollectionSetCandidateInfo* ci1, G1CollectionSetCandidateInfo* ci2) {
+int G1CollectionSetCandidateInfo::compare_gc_efficiency(G1CollectionSetCandidateInfo* ci1, G1CollectionSetCandidateInfo* ci2) {
```

Comment:
Maybe make this name a bit more distinct than above `compare_gc_efficiency`, i.e. `compare_region_gc_efficiency` vs. `group_efficiency`.
Maybe however the different parameters are fine to distinguish them.